lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -228,42 +228,40 @@ def _get_saved_grade(self, subsection, course_structure, course):  # pylint: dis
                 subsection_grade = SubsectionGrade(subsection)
                 subsection_grade.load_from_data(model, course_structure, self._scores_client, self._submissions_scores)
                 log.warning(
-                    "Loaded saved grade for course with id {0}, version {1}, subsection {2}. grade: {3}".format(
+                    "Persistent Grades: Loaded saved grade for course with id {0}, version {1}, subtree edited on {2}, "
+                    "grade: {3}".format(
                         unicode(course.id),
-                        unicode(course.course_version),
-                        unicode(subsection.location),
-                        subsection_grade.grade_data_as_unicode()
+                        unicode(getattr(course, 'course_version', None)),
+                        unicode(course.subtree_edited_on),
+                        unicode(subsection_grade)
```

Comment:
Also, the explicit `unicode()` calls are unnecessary.  If the template string is a unicode object, that's what format will do by default.

Demo: 

```python
>>> class Foo(object):
...     def __str__(self):
...         return 'bytes'
...     def __unicode__(self):
...         return u'text'
... 
>>> print ''.format(Foo())

>>> print '{}'.format(Foo())
bytes
>>> print u'{}'.format(Foo())
text
```